src/components/StudioTopbar/StudioTopbar.style.tsx

```diff
@@ -0,0 +1,180 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { Link } from 'react-router-dom'
+import { breakpoints, colors, sizes, transitions, typography, zIndex } from '@/shared/theme'
+import { TopbarBase, Avatar, Text, Button, TOP_NAVBAR_HEIGHT, TopbarBaseProps } from '@/shared/components'
+
+type CommonStudioTopbarProps = {
+  isActive?: boolean
+}
+
+export const StyledTopbarBase = styled((props: TopbarBaseProps) => <TopbarBase {...props} />)`
```

Comment:
any reason not to just do `styled(TopbarBase)`?

Reply:
Got an error `Error: You are trying to create a styled element with an undefined component. You may have forgotten to import it.`

Went back to `styled(TopbarBase)` but had to use relative import path for that component

Reply:
Yep, that's a known issue